python/tvm/exec/popen_worker.py

```diff
@@ -69,6 +79,7 @@ def _cancel_run(status):
         lock.release()
 
     while True:
+        reader = get_reader()
```

Comment:
I am a little concerned by the fact that we are creating file descriptor each time in an infinite loop. If this is a memory leak occurred within a subprocess, would this [issue](https://bugs.python.org/issue12650) more relevant?

Reply:
fdopen means open a file descriptor, not creating one

Reply:
Offline synced with @zxybazh and this fix seems to only re-opens the same file descriptor each time the popenworker is reused. @junrushao1994 Thanks for the clarification.

Reply:
Offline synced with @zxybazh and this fix seems to only re-opens the same file descriptor each time the popenworker is reused. @junrushao1994 Thanks for the clarification.